presubmit-service/check_if_pr_can_cherry_pick.sh

```diff
@@ -0,0 +1,66 @@
+#! /bin/bash
+
+# The organization containing the repo to be checked against, e.g. angular
+ORG=$1
+# The repo to be checked against, e.g. material2
+REPO=$2
+# The PR number to retrieve the patch files for, e.g. 1234
+PR=$3;
+# The branch in the repo to check for cherry-picking onto, e.g. 6.1.x
+BRANCH=$4;
+# The fullname of the repository to be checked against, e.g. angular/material2
+FULL_REPO="$ORG/$REPO"
```

Comment:
Prefer using the brace syntax for variables rather than just `$` by itself, e.g.
```sh
full_repo_name="${org}/${repo}"
```